package.json

```diff
@@ -22,6 +22,7 @@
         "adaptivecards-reactnative": "^2.2.11",
         "adaptivecards-templating": "^2.1.0",
         "babel-prettier-parser": "^0.10.0",
+        "botbuilder-dialogs-adaptive": "^4.14.1-preview",
```

Comment:
Can we use stable version of this library?

Reply:
ok, I will use stable version instead. Thanks.